content/en/continuous_integration/setup_tests/junit_upload.md

```diff
@@ -118,6 +116,11 @@ This is the full list of options available when using the `datadog-ci junit uplo
 **Example**: `team:backend`<br/>
 **Note**: Tags specified using `--tags` and with the `DD_TAGS` environment variable are merged. If the same key appears in both `--tags` and `DD_TAGS`, the value in the environment variable `DD_TAGS` takes precedence.
 
+`--logs` **(beta)**
+: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` will be loaded. Logs from elements inside a `<testcase>` are automatically connected to test.<br/>
```

Comment:
```suggestion
: Enable forwarding content from the XML reports as [Logs][7]. The content inside `<system-out>`, `<system-err>`, `<failure>` will be sent to the Logs system. Logs from elements inside a `<testcase>` are automatically linked to the test.<br/>
```

Reply:
I used the word connected because of https://docs.datadoghq.com/tracing/connect_logs_and_traces/ but good suggestions for the others I'll make the changes